packages/extension/src/pages/sign/utils/agoric/display-amount.ts

```diff
@@ -0,0 +1,64 @@
+const CONVENTIONAL_DECIMAL_PLACES = 2;
+const MAX_DECIMAL_PLACES = 100;
+
+const roundToDecimalPlaces = (rightOfDecimalStr = "", decimalPlaces = 0) => {
```

Comment:
It looks like you can replace functions in this file with `Dec` or `Int` from @keplr-wallet/unit. See if you can use `Dec` or `Int`

Reply:
Thanks, I was able to get something simple working with `CoinPretty`. I'm passing an empty string as `coinMinimalDenom` because it's irrelevant, hopefully that's okay for this use-case.